workman/src/test/java/org/duracloud/mill/bit/BitIntegrityReportTaskProcessorTest.java

```diff
@@ -124,8 +125,12 @@ public void testExecute() throws Exception {
                                                eq(storeId),
                                                eq(spaceId))).andReturn(it);
         final Capture<InputStream> capture = new Capture<>();
-
-        this.store.addContent(eq("x-duracloud-admin"),
+        String reportSpace = "x-duracloud-admin";
+        expect(this.store.getSpaces()).andReturn(Arrays.asList("aspace").iterator());
+        this.store.createSpace(reportSpace);
+        
```

Comment:
Shouldn't there be an expectLastCall() method here? 
(There's also an unnecessary newline here.)